core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -522,6 +529,13 @@ trait DeltaSQLConfBase {
       .booleanConf
       .createWithDefault(true)
 
+  val DATA_SKIPPING_STRING_PREFIX_LENGTH =
+    SQLConf.buildConf("spark.databricks.io.skipping.stringPrefixLength")
+      .internal()
```

Comment:
same here: probably shouldn't be internal, eh?

Reply:
same response as above.

Reply:
add a comment regarding why the prefix is different from other delta confs.